### PR TITLE
JBPM-7312: Stunner - Marshaller tests for End Events are failing

### PR DESCRIPTION
**Pull request by evacchi**

### CODE REVIEW

**Comment by evacchi** [comment]
Jenkins please retest this

**Comment by evacchi** [comment]
:(

Jenkins please retest this

File: kie-wb-common-stunner/kie-wb-common-stunner-sets/kie-wb-common-stunner-bpmn/kie-wb-common-stunner-bpmn-backend/src/test/java/org/kie/workbench/common/stunner/bpmn/backend/service/diagram/marshalling/events/EndEvent.java [hunk 1]
```diff
@@ -54,7 +54,7 @@
     public static List<Object[]> marshallers() {
         return Arrays.asList(new Object[][]{
                 // New (un)marshaller is disabled for now due to found incompleteness
```
Comment (hasys) [inline kie-wb-common-stunner/kie-wb-common-stunner-sets/kie-wb-common-stunner-bpmn/kie-wb-common-stunner-bpmn-backend/src/test/java/org/kie/workbench/common/stunner/bpmn/backend/service/diagram/marshalling/events/EndEvent.java]: Comment can be removed now.

**Comment by evacchi** [comment]
@hasys done
@romartin you know what to do!

**Pull request merged**